frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -858,6 +863,14 @@ export const surveyLogic = kea<surveyLogicType>([
                     reloadAllSurveyResults()
                 }
             },
+            setShowSkipped: () => {
+                // Reload single choice results for all questions that are of type SingleChoice
+                values.survey.questions.forEach((question, index) => {
+                    if (question.type === SurveyQuestionType.SingleChoice) {
```

Comment:
will this happen only for single choice?

Reply:
yes. this is not really a problem with the query per say, it's mostly with the bar chart visualization, where it consider it for the label even when the value is `null`. 

i chagned the query now to ignore null values